README.md

```diff
@@ -40,29 +28,75 @@ In the source folder of your application, for Nano S:
 
 ```bash
 $ # docker can be replaced with podman or buildah without sudo
-$ sudo docker run --rm -ti -v "$(realpath .):/app" ledger-app-builder:latest
-root@656be163fe84:/app# make
+$ sudo docker run --rm -ti -v "$(realpath .):/app" --user $(id -u $USER):$(id -g $USER) ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder:latest
+root@656be163fe84:/app# BOLOS_SDK=$NANOS_SDK make
 ```
 
 The Docker image includes the [Clang Static Analyzer](https://clang-analyzer.llvm.org/), which can be invoked with:
 
 ```bash
 $ # docker can be replaced with podman or buildah without sudo
-$ sudo docker run --rm -ti -v "$(realpath .):/app" ledger-app-builder:latest
-root@656be163fe84:/app# make scan-build
+$ sudo docker run --rm -ti -v "$(realpath .):/app" --user $(id -u $USER):$(id -g $USER) ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder:latest
+root@656be163fe84:/app# BOLOS_SDK=$NANOS_SDK make scan-build
 ```
 
-For Nano X, specify the `BOLOS_SDK` environment variable before building your app:
+For Nano X, specify the `BOLOS_SDK` environment variable before building your app with the pre-defined `$NANOX_SDK` variable:
 
 ```bash
 $ # docker can be replaced with podman or buildah without sudo
-$ sudo docker run --rm -ti -v "$(realpath .):/app" ledger-app-builder:latest
+$ sudo docker run --rm -ti -v "$(realpath .):/app" --user $(id -u $USER):$(id -g $USER) ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder:latest
 root@656be163fe84:/app# BOLOS_SDK=$NANOX_SDK make
 ```
 
-For Nano S+, specify the `BOLOS_SDK` environment variable before building your app:
+For Nano S+, specify the `BOLOS_SDK` environment variable before building your app with the pre-defined `$NANOSP_SDK` variable:
 
 ```bash
 $ # docker can be replaced with podman or buildah without sudo
-$ sudo docker run --rm -ti -v "$(realpath .):/app" ledger-app-builder:latest
+$ sudo docker run --rm -ti -v "$(realpath .):/app" --user $(id -u $USER):$(id -g $USER) ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder:latest
 root@656be163fe84:/app# BOLOS_SDK=$NANOSP_SDK make
+```
+
+## Load the app on a physical device
+
+:warning: Only Nano S and Nano S+ devices allow application side-loading. This section will not work with a Nano X.
+
+To load the app from the container, you will need additional docker arguments in order to allow Docker to access your USB port.
+Your physical device must be connected, unlocked and the screen showing the dashboard (not inside an application).
+Use the following docker command:
+
+```bash
+$ sudo docker run --rm -ti  -v "$(realpath .):/app" --privileged -v "/dev/bus/usb:/dev/bus/usb" --user $(id -u $USER):$(id -g $USER) ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder:latest
```

Comment:
I would recommend always using the usb option, there is no downside in doing so

Reply:
Except for the clarity of the `README`. I'm not keen on over-informing a reader with options not useful yet. The USB option is useful for loading only. This README reflects that. Once the user understand this, he is free to add it anywhere it pleases him.